kivymd/uix/label.py

```diff
@@ -319,6 +320,15 @@ def __init__(self, **kwargs):
         self.update_font_style()
         self.on_opposite_colors(None, self.opposite_colors)
 
+    def check_font_styles(self):
+        if self.font_style not in list(self.theme_cls.font_styles.keys()):
+            raise ValueError(
+                "MDLabel.font_style is set to an invalid option '{}'. "
+                "Must be one of: {}".format(
+                    self.font_style, list(self.theme_cls.font_styles)
+                )
+            )
+
```

Comment:
This seems like a good idea, but it should be a bind function.

We don't know if the developer will be changing the MDLabel property on the run.

what i believe it must be done is to bind directly to theme_cls.font_styles, so every time it changes, all the MDlabel class updates it's `font_style`  **option** attribute

Reply:
Currently, this is a non-issue imo, as whenever `self.theme_cls.font_styles` updates, `self.font_style` will also accept whatever that `self.theme_cls.font_style` has, since it iterates over its dict keys, meaning if `self.theme_cls.font_style` is updated, `self.font_style` will not raise an error if it has been added to `self.theme_cls.font_style`.

However, I do propose adding a check fn in the `self.update_font_style()` fn such that instead of raising a `key_error` when an unknown font_style is set, it will raise `ValueError: MDLabel.font_style is set to an invalid option` etc.